hail/python/test/hail/linalg/test_linalg.py

```diff
@@ -386,6 +386,8 @@ def test_special_elementwise_ops(self):
 
         self._assert_close(m ** 3, nm ** 3)
         self._assert_close(m.sqrt(), np.sqrt(nm))
+        self._assert_close(m.ceil(), np.ceil(nm))
+        self._assert_close(m.floor(), np.floor(nm))
```

Comment:
with the current values this doesn't actually test anything. Could you update them to some decimal values?